design-tokens/tokens/layer3/base.json

```diff
@@ -13,6 +13,22 @@
             "value": "150px",
             "type": "sizing"
           }
+        },
+        "width": {
+          "default": {
+            "value": "37.5rem",
+            "type": "sizing"
+          },
+          "wide": {
+            "value": "62.5rem",
+            "type": "sizing"
+          }
+        },
+        "maxHeight": {
+          "default": {
+            "value": "80%",
```

Comment:
This is wrong, it should be `80vh` instead.

Reply:
Fixed

Reply:
Yes, fixed ✅